packages/react/src/lib/felaRenderer.tsx

```diff
@@ -37,7 +37,10 @@ const createRendererConfig = (options: any = {}) => ({
   ...(options.rendererId ? { rendererId: options.rendererId } : {}),
 })
 
-export const felaRenderer: Renderer = createRenderer(createRendererConfig())
+// { rendererId: 'default' } is a temporary workaround for https://github.com/stardust-ui/react/issues/948#issuecomment-466404630
+export const felaRenderer: Renderer = createRenderer(
+  createRendererConfig({ rendererId: 'default' }),
+)
```

Comment:
I am not fan of this solution, but the fix in Fela is not ready. Can we mark this as todo to allow find this item easier in the future?